src/main/java/bio/terra/service/auth/iam/IamResourceType.java

```diff
@@ -36,4 +37,32 @@ static IamResourceType fromValue(String text) {
     }
     return null;
   }
+
+  public static IamResourceType toIamResourceType(IamResourceTypeEnum apiEnum) {
```

Comment:
It's a matter of perspective, but I prefer `from()` for methods that are converting from some other type to the current type. I would use `to()` for a method that converts from the current type to another type. So I would call this `fromResourceEnum` or `fromEnum`.
```suggestion
  public static IamResourceType fromEnum(IamResourceTypeEnum apiEnum) {
```

Reply:
I like this suggestion, `IamResourceType.fromEnum` reads more logically.